apps/charterafrica/src/payload/fields/post.js

```diff
@@ -0,0 +1,76 @@
+import BlogContent from "../blocks/BlogContent";
+import MediaBlock from "../blocks/MediaBlock";
+import VideoEmbed from "../blocks/VideoEmbed";
+
+import richText from "./richText";
+import slug from "./slug";
+
+const setPublishedOn = ({ value }) => {
+  return new Date(value);
+};
+
+const postFields = [
+  {
+    name: "title",
+    label: {
+      en: "Title",
+      fr: "Titre",
+      pt: "Título",
+    },
+    type: "text",
+    localized: true,
+    required: true,
+  },
+  slug(),
+  {
+    name: "authors",
+    required: true,
+    type: "relationship",
+    relationTo: "author",
+    hasMany: true,
+  },
+  {
+    name: "coverImage",
+    label: {
+      en: "Cover Image",
+      pt: "Imagem de capa",
+      fr: "Image de couverture",
+    },
+    type: "upload",
+    relationTo: "media",
+    required: true,
+    filterOptions: {
+      mimeType: { contains: "image" },
+    },
+  },
+  richText({
+    name: "excerpt",
```

Comment:
👇🏽 is how Excerpt look right now.
![Screenshot 2023-02-24 at 13-26-24 Creating - News - Payload](https://user-images.githubusercontent.com/1779590/221155712-e8b762a1-601e-4820-879d-8347291dbdab.png)

That's how full content should look like. excerpt is like summary; shouldn't contain heading, images or other uploads, etc. Probably should just contain `leaves` i.e. bold, italic, etc.